toolkit/tools/scheduler/scheduler.go

```diff
@@ -387,6 +388,19 @@ func buildAllNodes(stopOnFailure, isGraphOptimized, canUseCache bool, packagesTo
 					}
 				}
 
+				if res.Node.Type == pkggraph.TypeBuild && res.WasDelta {
```

Comment:
>if res.Node.Type == pkggraph.TypeBuild && res.WasDelta { [](http://example.com/codeflow?start=4&length=56)

What's the reason to run this logic even if `res.Err != nil`? Making sure this is by design.